app/models/episode.rb

```diff
@@ -208,6 +208,15 @@ def copy_media(force = false)
     images.each { |i| i.copy_media(force) }
   end
 
+  def apple_mark_for_reupload!
+    apple_podcast_deliveries.destroy_all
+  end
+
+  def publish!
+    apple_mark_for_reupload!
```

Comment:
Just noting that this method will get called _even if_ `episode.published_at.nil?` (the ep isn't published yet).

In that case ... nothing bad would happen, right?  Just a no-op, since it would have no deliveries.

Reply:
Yep, it would be a no-op. And since we only pull episodes from Feed#feed_episodes we should only get the episodes that are published in the RSS.